keyboards/kingly_keys/little_foot/little_foot.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "quantum.h"
+
+#define XXX KC_NO
+
+#define SPLITSPACEBASE( \
```

Comment:
This should be:
```suggestion
#define LAYOUT_split_space_base( \
```

Generally, it should always start with `LAYOUT`, and the rest should be lower case.  

I'm sure that @noroadsleft will have more input on this.